src/popup/popup.js

```diff
@@ -3,6 +3,7 @@ import store from '../store';
 import App from './App';
 import router from './router';
 import { i18n } from './utils/i18nHelper';
+import initEnv from '../lib/initEnv'
```

Comment:
Actually this is needed here so I am returning it - previously removed in https://github.com/waellet/waellet/pull/344/commits/8ea65b1039707a05fdb9f98133b44f6920f6abe6